packages/waas/src/payloads/signers/index.ts

```diff
@@ -0,0 +1,28 @@
+import { newSECP256K1RandomSigner, newSECP256K1Signer } from "./secp256k1";
+import { newSECP256R1Signer, newSECP256R1RandomSigner } from "./secp256r1";
+
+export type PayloadSigner = {
+  privateKey: string
+
+  verifier(): Promise<string>
+  signMessage(message: string | Uint8Array): Promise<string>
+}
```

Comment:
I think we'll also want to add the method `verify(message: string, signature: string): Promise<boolean>` or similar, to complete this interface of signing and verifying for a pub/priv signing algorithm

Reply:
I did not implement the verify method because we never will be using this in this context. It might be useful for unit tests though. So I guess I will do that.

Reply:
yes, its easy to do, and for completion we should do it